pilot/proxy/envoy/testdata/cds-ssl-context-optout.json.golden

```diff
@@ -159,6 +159,42 @@
      "verify_subject_alt_name": []
     }
    },
+   {
+    "name": "out.httpbin.default.svc.cluster.local|http",
+    "service_name": "httpbin.default.svc.cluster.local|http",
+    "connect_timeout_ms": 1000,
+    "type": "strict_dns",
+    "lb_type": "round_robin",
+    "hosts": [
+     {
+      "url": "tcp://httpbin.default.svc.cluster.local:80"
+     }
+    ],
+    "ssl_context": {
+     "cert_chain_file": "/etc/certs/cert-chain.pem",
+     "private_key_file": "/etc/certs/key.pem",
+     "ca_cert_file": "/etc/certs/root-cert.pem",
+     "verify_subject_alt_name": []
+    }
```

Comment:
here is the other problem.. You cannot use istio auth for kube external services, because there is no istio-egress proxy. Traffic has to go directly from sidecar to external service.

Reply:
Opened https://github.com/istio/istio/issues/2842 to fix it in a separate PR.